src/browser/modules/Editor/Monaco.tsx

```diff
@@ -171,7 +171,9 @@ const Monaco = forwardRef<MonacoHandles, MonacoProps>(
           selectionHighlight: false,
           value,
           wordWrap: 'on',
-          wrappingStrategy: 'advanced'
+          wrappingStrategy: 'advanced',
```

Comment:
A little too much scope creep to be changing monaco settings for a PR about buttons imo, let's try to do less of this in the future... :-)

Reply:
Yeah, it changed from just buttons to a full UI-refresh.